webanno-diag/src/main/java/de/tudarmstadt/ukp/clarin/webanno/diag/CasDoctorUtils.java

```diff
@@ -34,22 +34,20 @@
     public static Set<FeatureStructure> collectIndexed(CAS aCas)
     {
         LowLevelCAS llcas = aCas.getLowLevelCAS();
-        Set<FeatureStructure> fses = new TreeSet<>((fs1, fs2) -> llcas.ll_getFSRef(fs1)
```

Comment:
I think not. The JVM bug seems to be related to using a handle of a method declared in a superclass to as serializable lambda instance. I don't think the comparator here is serialized.